bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,328 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="evcc"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="device">
+
+		<label>evcc installation</label>
+		<description>A running evcc installation</description>
+
+		<channel-groups>
+			<channel-group id="general" typeId="general"/>
+			<channel-group id="loadpoint0" typeId="loadpoint"/>
+			<channel-group id="loadpoint1" typeId="loadpoint"/>
+			<channel-group id="loadpoint2" typeId="loadpoint"/>
+			<channel-group id="loadpoint3" typeId="loadpoint"/>
+			<channel-group id="loadpoint4" typeId="loadpoint"/>
+			<channel-group id="loadpoint5" typeId="loadpoint"/>
+			<channel-group id="loadpoint6" typeId="loadpoint"/>
+			<channel-group id="loadpoint7" typeId="loadpoint"/>
+			<channel-group id="loadpoint8" typeId="loadpoint"/>
+			<channel-group id="loadpoint9" typeId="loadpoint"/>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="url" type="text" required="true">
+				<context>network-address</context>
+				<label>URL</label>
+				<description>URL of evcc web UI, e.g. https://demo.evcc.io</description>
+				<default>https://demo.evcc.io</default>
```

Comment:
Does it make sense to default a demo URL? Parameter is required, so by leaving a default, user will not be forced to input anything meaningful.